packages/tocco-ui/src/FormattedValue/FormattedValue.stories.js

```diff
@@ -1,146 +1,188 @@
 /* eslint-disable react/prop-types */
 import React from 'react'
-import {storiesOf} from '@storybook/react'
-import {withKnobs, date, boolean, number, object, text, select} from '@storybook/addon-knobs'
-import moment from 'moment'
 
 import FormattedValue from './'
 import {storybookHtmlMarkup} from '../util/storybookHtmlMarkup'
+import {map} from './typeFormatterProvider'
+export default {
```

Comment:
sort elements alphabetically that it is easier to find in the UI-menu (same for editableValue)